src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -261,6 +261,24 @@ final class SecureStore: Store {
 	#endif
 }
 
+extension SecureStore {
+
+	var warnOthersNotificationOneTimer: Int {
```

Comment:
Use `TimeInterval` naming and data type

Reply:
Thanks 👍 . Totally overseen this. Refactored